galley/pkg/server/components/processing.go

```diff
@@ -26,24 +26,28 @@ import (
 	"google.golang.org/grpc/keepalive"
 	grpcMetadata "google.golang.org/grpc/metadata"
 
+	mcp "istio.io/api/mcp/v1alpha1"
```

Comment:
These changes are a result of `git mv processing2.go processing.go`. No meaningful changes were made to processing2.go before or after the rename.